### ISSUE TITLE
Rano'erek/Dredger needs less crew than Palavret

### ISSUE THREAD

**Issue by ThrawnCA** [endless-sky/endless-sky#8023]
### Problem Description

The Rano'erek, IIUC, is meant to be the Palavret's bigger and badder cousin. However, its minimum crew requirement is actually less than the Palavret, though it has far more bunks available.

This is obviously quite helpful for the player, should they manage to capture one (it's an excellent flagship), but it seems odd, especially for a Korath ship, and double especially when it means that the Rano'erek is occasionally easier to capture than a Palavret.

### Related Issue Links

None found.

### Desired Solution

The Rano'erek seems like it should have a higher minimum crew requirement.

### Alternative Approaches

The Rano'erek could use microbot defences to make it harder to capture. This seems logical if it's designed to have a low crew requirement; it's a valuable ship that they would want to protect, so adding a few anti-intrusion systems would make sense.

**Comment by the2nake** [comment]
Palavrets and Rano'ereks are equally difficult to capture (i.e. not at all) because they are bounty ships. After disabling them, a player can simply board, kill off some of the crew, retreat, land, hire some more goons, and board the ship again to capture it. In this manner, I was able to capture a Rano'erek with an Albatross fairly consistently.

**Comment by ThrawnCA** [comment]
> Palavrets and Rano'ereks are equally difficult to capture (i.e. not at all) because they are bounty ships. After disabling them, a player can simply board, kill off some of the crew, retreat, land, hire some more goons, and board the ship again to capture it. In this manner, I was able to capture a Rano'erek with an Albatross fairly consistently. Thus, making them slightly harder to capture normally doesn't really change the game's balance (they're already both very easy to obtain)

Yes, and no. I've used that trick myself, but I wouldn't say it makes this issue irrelevant.

- That trick applies to the Remnant missions, but not to other contexts like the random spawns in the Korath systems.
- Crew in a disabled ship will reset to the minimum requirement after landing and relaunching, so the minimum still matters.
- Minimum crew requirement still affects ongoing maintenance costs.

**Comment by Saugia** [comment]
@ravenshining

**Comment by the2nake** [comment]
>     * That trick applies to the Remnant missions, but not to other contexts like the random spawns in the Korath systems.

I suppose this is true, though again, once you have obtained a ship from the Remnant missions, it should be easy to capture one of the random spawns using it.

>     * Crew in a disabled ship will reset to the minimum requirement after landing and relaunching, so the minimum still matters.

If this mechanic stays in place, sure. Crew in a disabled ship resetting doesn't really make sense, though. Where are the extra crew members coming from? I'd get it if it resets when the ship is repaired, but I think if the ship stays disabled it should not regenerate crew (it's not like the crew members are birthing children in a day, after all)

>     * Adding automated defences would make the trick much harder.

I disagree. The trick would still be fairly easy, because you only have to attack once. You probably won't kill most of your crew members in one attack. Because automated defences don't impair your ability to defend, it would be pretty easy to run and land.

>     * Minimum crew requirement still affects ongoing maintenance costs.

This is true. I find maintenance cost to be mostly irrelevant at this stage of the game, though (disabling a single mission-spawned Palavret and looting it gives millions of credits).

**Comment by ThrawnCA** [comment]
> I suppose this is true, though again, once you have obtained a ship from the Remnant missions, it should be easy to capture one of the random spawns using it.

It gets less easy if the minimum crew level in the target is higher, and much less easy if they have defence systems.

> > * Crew in a disabled ship will reset to the minimum requirement after landing and relaunching, so the minimum still matters.
> > ```
> 
> If this mechanic stays in place, sure. Crew in a disabled ship resetting doesn't really make sense, though. Where are the extra crew members coming from? I'd get it if it resets when the ship is repaired, but I think if the ship stays disabled it should not regenerate crew (it's not like the crew members are birthing children in a day, after all)

True, but as it stands, there would be a difference.

> The trick would still be fairly easy, because you only have to attack once. You probably won't kill most of your crew members in one attack. Because automated defences don't impair your ability to defend, it would be pretty easy to run and land.

But if the ship is hard enough to crack, it might mean having to make a third try or even a fourth. Especially since automated defences keep working even if you whittle down the crew. Every day you take to bring the ship down means more salaries, and more chances for another mission objective ship to jump in and cause you to accidentally blow up the first one, and more sense that you've had to put in a real effort to succeed. So it's meaningful.

**Comment by Zitchas** [comment]
> If this mechanic stays in place, sure. Crew in a disabled ship resetting doesn't really make sense, though. Where are the extra crew members coming from? I'd get it if it resets when the ship is repaired, but I think if the ship stays disabled it should not regenerate crew (it's not like the crew members are birthing children in a day, after all)

My personal head canon is that, despite what the labels says, we're not necessarily *killing* people during boarding combat, just injuring/disabling enough that they *appear* to be dead; and that while we go land and a day passes, they're busy doing whatever passes for first aid and emergency medical treatment in the 31st century and saving a bunch of people.

I know it doesn't completely make sense, but it would cover at least some of it.

**Comment by ravenshining** [comment]
> The Rano'erek, IIUC, is meant to be the Palavret's bigger and badder cousin. 

I think it was *conceived* that way, but we've been treating it more as a freighter.  

> The Rano'erek could use microbot defences

We were originally keeping those out of Human space when they were considered an `atrocity`, but now that they're just `illegal`... sure, we could revisit that.

> Palavrets and Rano'ereks are equally difficult to capture (i.e. not at all) because they are bounty ships.

Yeeeeeaahh the cappability of bounties really feels like a *bug* to me more than a *feature*.  Personally I think all bounties should have time limits at *least* (you're not the only bounty hunter in the galaxy), or be considered "killed" if they are disabled when you land, but that's a separate issue.

**Comment by Zitchas** [comment]
> Yeeeeeaahh the cappability of bounties really feels like a _bug_ to me more than a _feature_. Personally I think all bounties should have time limits at _least_ (you're not the only bounty hunter in the galaxy), or be considered "killed" if they are disabled when you land, but that's a separate issue.

I fully agree on this one. I'm really not a fan of the "once disabled you can loot the ship, go sell the loot, then come back to loot it a few more times" thing. The Remnant themselves, for instance, would step in once the ship is disabled, they're not going to just leave it floating around forever. 

In an ideal world, these would change over time. 
- Player gets the mission, and the bounty as tagged as "player's going to deal with it" so it is left alone.
- A week goes by and the ship is still not-disabled? the "player's going to deal with it" tag is removed and Remnant start attacking it.
- If it *is* disabled, then the Remnant thank and pay the player, and the ship goes away.

**Comment by warp-core** [comment]
> Personally I think all bounties should have time limits at least (you're not the only bounty hunter in the galaxy)

I think most/all the human ones do.

> And even less so the iterative boarding attacks thing for capturing.

This isn't really a thing anymore and hasn't been for a long time.
The lowest the ships crew can be after the player lands and takes off again is its required crew.

**Comment by Zitchas** [comment]
It's still useful for whittling off the extra crew; but yeah, just taking a glance at the job board, all the bounty missions I can see right now have time limits on them.

**Comment by Zitchas** [comment]
Generally speaking, I dislike timelimits because I enjoy a more relaxed feel to the game. which is why I haven't implemented time limits on the Remnant ones so far. Looking at the documentation for the NPC section, it looks like it is *almost* possible to have it set an attribute to indicate when the NPC is disabled instead of destroyed. Alternatively, I could just set the mission to request it to be disabled, and have them pay a higher bounty if it is disable and a lower one if it is destroyed. That'd be fairly straight forward, and then the ship would dissapear more promptly too. Not 100% I can do it, but it seems like a possibility...

**Comment by ThrawnCA** [comment]
> Generally speaking, I dislike timelimits because I enjoy a more relaxed feel to the game. which is why I haven't implemented time limits on the Remnant ones so far.

Time limits do make sense, however, for bounty missions on ships that are nosing around.

Completing the mission once the ship is disabled and taking it off the board makes sense, too - especially for the Remnant, who love looting disabled Korath ships.

**Comment by ravenshining** [comment]
Perhaps also of note - during the planning of the Korath expansion, we had the concept of what we called "quality of life" outfits - things like nurseries, aquaponics, and machine shops that wouldn't really be practical for a normal starship, but which would be absolutely essential to the Korath.  These would be unplunderable and come with high crew requirements, and thus would inflate the crew size of Korath ships we deemed needed to be made more difficult to capture or which did not make sense to be unpopulated (the latter reason could apply to almost all of them, really), and therefore address this issue.

They might also be mildly useful to a player with a large vessel or fleet - having a botanical garden on your ship could attract high-paying passengers, or an aquaponics bay could produce 1 tonne of food per day - but probably not worth having for most, particularly the nursery which may offer no benefit whatsoever unless you either like pissing off Korath, or you are working for them and have a job that involves taking care of their young.

**Comment by Zitchas** [comment]
I am very much looking forward to seeing all those "Quality of Life (for a civilization living in space)" outfits. Story ideas abound!

**Comment by ThrawnCA** [comment]
> I am very much looking forward to seeing all those "Quality of Life (for a civilization living in space)" outfits.

That does sound cool.

On the immediate issue, does anyone else (thanks @ravenshining ) have an opinion about adding microbot defences as standard-issue on the Rano'erek?

**Comment by TheGiraffe3** [comment]
Is there a consensus on what needs to happen?

**Comment by Quantumshark** [comment]
The original point raised isn't really a problem and isn't going to be changed, so I don't think this issue needs to stay open.

Some of the other ideas raised here are good, but they should be opened in new issues if anyone wants to pursue them.
